src/modules/accommodations/entities/wishlist.entity.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Unique,
+} from 'typeorm';
+import { Accommodation } from './accommodation.entity';
+import { User } from 'src/modules/users/entities/user.entity';
+
+@Entity({ name: 'accommodation_like' })
+@Unique(['user', 'accommodation'])
+export class Wishlist {
+  @PrimaryGeneratedColumn('uuid') id: string;
```

Comment:
I think you reformat this class and put some empty spaces. it is a little bit hard to read.